dev/com.ibm.ws.jdbc_fat_db2/publish/servers/com.ibm.ws.jdbc.fat.db2/server.xml

```diff
@@ -108,6 +107,41 @@
     <jdbcDriver libraryRef="DB2Lib"/>
     <properties.db2.jcc databaseName="${DB2_DBNAME}" portNumber="${DB2_PORT}" user="${DB2_USER}" password="${DB2_PASS}" />
   </dataSource>
+  
+  <!-- 
+    Verify client re-route config
+  -->
+  <dataSource id="ds-client-reroute" jndiName="jdbc/ds-client-reroute">
+    <jdbcDriver libraryRef="DB2Lib"/>
+    <properties.db2.jcc serverName="fake.host.name" portNumber="1234" 
```

Comment:
```suggestion
    <properties.db2.jcc serverName="host.name.invalid" portNumber="1234" 
```
`.invalid` is a reserved top level domain for things like this. `fake.host.name` could be a real domain name